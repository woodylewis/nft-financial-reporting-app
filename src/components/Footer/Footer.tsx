import * as React from 'react';

import Grid from '@mui/material/Grid';
import Typography from '@mui/material/Typography';
import Hidden from '@mui/material/Hidden';

import { StringList } from '../../constants/StringList';

const Footer = () => {
  return (
    <Grid container mt={2}>
      <Hidden only={['xs']}>
        <Grid item xs={12} textAlign='center'>
          <Typography variant='caption' color='textSecondary'>
            {StringList.FOOTER_COPYRIGHT}
            <br />
            {StringList.FOOTER_TRADEMARK}
          </Typography>
        </Grid>
      </Hidden>
      <Hidden only={['sm', 'md', 'lg', 'xl']}>
        <Grid item xs={12} textAlign='center'>
          <Typography variant='caption' color='textSecondary'>
            {StringList.MOBILE_FOOTER_COPYRIGHT_ONE}
            <br />
            {StringList.MOBILE_FOOTER_COPYRIGHT_TWO}
            <br />
            {StringList.MOBILE_FOOTER_TRADEMARK}
          </Typography>
        </Grid>
      </Hidden>
    </Grid>
  );
};

export default Footer;
