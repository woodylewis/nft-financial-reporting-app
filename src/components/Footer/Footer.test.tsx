import * as React from 'react';
import { render, cleanup } from '@testing-library/react';
import Footer from './Footer';

describe('Footer component', () => {
  afterEach(cleanup);

  it('should match snapshot', () => {
    const { asFragment } = render(<Footer />);
    expect(asFragment).toMatchSnapshot();
  });
});
