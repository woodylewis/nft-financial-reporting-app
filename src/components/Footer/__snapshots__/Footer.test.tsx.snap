// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component should match snapshot 1`] = `[Function]`;
